### DISCUSSION TITLE
Set styling on DataGrid Header Row?

### DISCUSSION THREAD

**Discussion by jmoerdyk** [Megabit/Blazorise#2965] [Q&A] [answered]
How does one set the background color on the Header Row for a DataGrid?   I see the `HeaderRowStyling` parameter in Intellisense, but I can't figure out  what values to set it, and I can't fing any documentation about it.

**Comment by David-Moreira** [comment, marked as answer]
Hello,
It is expecting an object of type `DataGridRowStyling`

**Usage Example:**
`HeaderRowStyling="new DataGridRowStyling() { Background = Background.Primary}"`
Or just bind a field or property of this same type with the configuration you want.

**Reply by jmoerdyk** [reply]
Many thanks

**Comment by apaulin-rbbn** [comment]
Is it possible to set the background color to a custom one? Something like #ddffdd or rgba(221, 255, 221, 1)

**Reply by stsrki** [reply]
Yes, it is the process, just a different property: `HeaderRowStyling="@(new DataGridRowStyling() { Style="color=rgba(221, 255, 221, 1);" })"`
